pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tarsier"
-version = "0.3.3"
+version = "0.3.4"
```

```suggestion
version = "0.4.0"
```

Theoretically this is a small breaking changes as we are now cleaning up after ourselves